bedrock/firefox/templates/firefox/win10-welcome.html

```diff
@@ -0,0 +1,140 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{ _('Choose the independent browser') }}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{ _('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.') }}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-win10-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_win10_welcome' %}
+{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Firefox + Windows 10. Perfect together.') }}{% endblock %}
+{% block body_id %}win10-welcome{% endblock %}
+{% block body_class %}mozID{% endblock %}
+
+{% block site_header %}
+<header id="masthead">
+  <div id="tabzilla">
+    <a href="{{ url('mozorg.home') }}">Mozilla</a>
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <header class="main-header">
+    <h1>{{ _('Firefox + Windows 10. Perfect together.') }}</h1>
+    <p class="tagline">
+      {{ _('We’ve designed Firefox to work seamlessly with your Windows 10 update, giving
+      you the same great browsing experience you’ve come to love, with the privacy, speed
+      and choice you’ve come to expect.') }}
+    </p>
+  </header>
+  <section class="firefox-default-cta hidden">
+    <h2>{{ _('Make Firefox your default in 3 easy steps:') }}</h2>
+    <ol>
+      <li>{{ _('Click the button below') }}</li>
+      <li>{{ _('Scroll down to “Web browser”') }}</li>
+      <li>{{ _('Choose Firefox') }}</li>
+    </ol>
+    <button id="set-default" type="button" class="button-flat-dark">{{ _('Let’s do it') }}</button>
+  </section>
+  <section class="firefox-learn-links">
```

Comment:
Should this not be using the hidden class as well, as apposed to `display: none;` in CSS?